ClosedXML/Excel/ConditionalFormats/IXLConditionalFormat.cs

```diff
@@ -140,8 +140,9 @@ public interface IXLConditionalFormat
         Boolean ShowIconOnly { get; }
         Boolean ShowBarOnly { get; }
         Boolean StopIfTrue { get; }
+        [Obsolete("Use Ranges instead")]
```

Comment:
I actually think we should keep `Range` forever, not make it obsolete. Most/many conditional formats will refer to only 1 range and this property will ease it a lot.

Reply:
OK